gradoop-examples/src/main/java/org/gradoop/examples/benchmark/grouping/GroupingBenchmark.java

```diff
@@ -0,0 +1,459 @@
+/*
+ * This file is part of Gradoop.
+ *
+ * Gradoop is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Gradoop is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Gradoop. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.gradoop.examples.benchmark.grouping;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.io.FileUtils;
+import org.apache.flink.api.common.ProgramDescription;
+import org.gradoop.examples.AbstractRunner;
+import org.gradoop.model.impl.LogicalGraph;
+import org.gradoop.model.impl.operators.grouping.Grouping;
+import org.gradoop.model.impl.operators.grouping.GroupingStrategy;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.CountAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.MaxAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.MinAggregator;
+import org.gradoop.model.impl.operators.grouping.functions.aggregation.PropertyValueAggregator;
+import org.gradoop.model.impl.pojo.EdgePojo;
+import org.gradoop.model.impl.pojo.GraphHeadPojo;
+import org.gradoop.model.impl.pojo.VertexPojo;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+
+/**
+ * A dedicated program for parametrized graph grouping benchmark.
+ */
+public class GroupingBenchmark extends AbstractRunner
+  implements ProgramDescription {
+
+  /**
+   * Option to declare path to input graph
+   */
+  public static final String OPTION_INPUT_PATH = "i";
```

Comment:
those constants can all be private cause they are only accessed from within this class